docs/workflow.rst

```diff
@@ -3,255 +3,182 @@
-also be important in some situations, so there is code to calculate it as well.
 
-Adding source density or background to observations
-===================================================
+These ASTs should be processed with the same code that was used to extract the
+source photometry.
 
-Create a new version of the observations that includes a column with the
-source density.  The new observation file includes only sources that have
-measurements in all bands (columns that match 'X_RATE').  In theory, sources
-without measurements in all bands is the result of non-overlapping observations.
-The BEAST is based on fitting sources with the same selection function,
-in this case measurements in all bands.
 
-A number of source density images are also created.  These include images
-that map the source density of objects with zero fluxes in different bands
-(or any band).
+*******************
+Edit/Split Catalogs
+*******************
 
-Command to create the observed catalog with source density column with
-a pixel scale of 5 arcsec using the 'datafile.fits' catalog.
+You may wish to remove artifacts from the photometry catalog.  If you do so, the
+same criteria must be applied to the AST catalog.
 
-  .. code-block:: console
+Commands to edit the files, both to remove flagged sources and eliminate sources
+that don't have full imaging coverage:
 
-     $ ./beast/tools/create_background_density_map.py sourceden -catfile datafile.fits --pixsize 5.
+.. code-block:: console
+
+   $ beast.tools.cut_catalogs phot_catalog_with_sourceden.fits phot_catalog_cut.fits \
+         --partial_overlap --region_file --flagged --flag_filter F475W
+   $ beast.tools.cut_catalogs ast_catalog.fits ast_catalog_cut.fits \
+         --partial_overlap --region_file --flagged --flag_filter F475W
 
-Split up observations by source density
----------------------------------------
 
 The observed catalog should be split into separate files for each source
 density.  In addition, each source density catalog is split into a set of
 sub files to have at most 'n_per_file' sources.  The sources are sorted by
 the 'sort_col' flux before splitting to put sources with similar brightness
 together.  This splitting into sub files sorted by flux allows for trimming
-the BEAST physics+observation model removing objects that are too bright
+the BEAST physics+observation model, removing objects that are too bright
 or too faint to fit any of the sources in the file.  In addition, this
 allows for running the BEAST fitting in parallel with each sub file
 on a different core.
 
-Command to create the the source density split files
+Command to split both the catalog and AST files by source density:
 
  .. code-block:: console
 
-    $ ./beast/tools/subdivide_obscat_by_source_density.py --n_per_file 6250 \
-             --sort_col F475W_RATE datafile_with_sourceden.fits
-
+    $ beast.tools.split_catalog_using_map.py phot_catalog_cut.fits ast_catalog_cut.fits \
```

Comment:
Need to covert commands to command line versions (see prev comments).

Reply:
resolved per discussion below